modules/migration/m1-payout-all-migration.js

```diff
@@ -0,0 +1,85 @@
+const models = require('../../models');
+const Utilities = require('../Utilities');
+
+const BATCH_SIZE = 15;
+
+/**
+ * Runs all pending payout commands
+ */
+class M1PayoutAllMigration {
+    constructor({
+        logger, blockchain, config, notifyError,
+    }) {
+        this.logger = logger;
+        this.config = config;
+        this.blockchain = blockchain;
+        this.notifyError = notifyError;
+    }
+
+    /**
+     * Run migration
+     */
+    async run() {
+        /* get all pending payouts */
+        let pendingPayOuts = await models.commands.findAll({
+            where: {
+                status: 'PENDING',
+                name: 'dhPayOutCommand',
+            },
+        });
+
+        if (pendingPayOuts.length === 0) {
+            this.logger.warn('No pending payouts.');
+            return;
+        }
+
+        const offerLimit = 60;
+        if (pendingPayOuts.length > offerLimit) {
+            const message = `Failed to complete payout for more that ${offerLimit}. Please contact support.`;
+            this.logger.error(message);
+            throw new Error(message);
+        }
+
+        const erc725Identity = Utilities.normalizeHex(this.config.erc725Identity);
+        while (pendingPayOuts.length > 0) {
+            const tempPending = pendingPayOuts.slice(0, BATCH_SIZE);
```

Comment:
tempPending is a redundant variable

Reply:
I'm not sure it is, tempPending stores the first `BATCH_SIZE` elements, and then those are cut off and the remainder is stored in `pendingPayOuts`.